news/9180.vendor.rst

```diff
@@ -0,0 +1,2 @@
+Upgrade resolvelib to 0.5.4 to fix error when an existing incompatibility is
```

Comment:
suggestion: break this into 2 fragments -- since the vendoring section is primarily for redistributors, and folks looking for whether a bug was fixed might be surprised to see this listed under vendoring?

A `resolvelib.vendor.rst` and a `9180.bugfix.rst` would be nice! :)